src/main.js

```diff
@@ -195,8 +199,10 @@ async function writeCss ({css, variant, bundleName, brandId, combinedChecksum, i
   if (brandId) cacheKey.push(brandId)
   cache.bundles_with_deps.update(joined(...cacheKey), {combinedChecksum, includedFiles})
   const filename = cssFilename({bundleName, variant, brandId, combinedChecksum})
+  const filename_without_checksum = cssFilename({bundleName, variant, brandId})
```

Comment:
I'm not super familiar with ES6 argument destructuring. Please correct me if I did it wrong :)